### PR TITLE
COMMERCE-7303 | COMMERCE-7305

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Review by ivicac — COMMENTED** [review]
@Commando18 Language files shouldn't be in this commit, they should be in a separate commit

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ImportCSVMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+		return PortletURLBuilder.create(
+			PortletProviderUtil.getPortletURL(
+				actionRequest, CommerceOrder.class.getName(),
+				PortletProvider.Action.EDIT)
+		).setMVCRenderCommandName(
+			"/commerce_open_order_content/view_commerce_order_importer_type"
+		).setBackURL(
+			ParamUtil.getString(actionRequest, "backURL")
+		).setParameter(
```
[the remark this answers is not in the document]
Comment (Commando18) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ImportCSVMVCActionCommand.java:160]: As far as I know, render command should be first, then backURL and redirect, then parameters and then windowState

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/upload/CSVUploadFileEntryHandler.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+		Company company = themeDisplay.getCompany();
+
+		return TempFileEntryUtil.addTempFileEntry(
+			company.getGroupId(), themeDisplay.getUserId(), _TEMP_FOLDER_NAME,
+			uniqueFileName, inputStream, contentType);
+	}
+
+	private boolean _exists(ThemeDisplay themeDisplay, String curFileName) {
+		try {
```
[the remark this answers is not in the document]
Comment (Commando18) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/upload/CSVUploadFileEntryHandler.java:90]: Yes, but I copied from portal

[the remark this answers is not in the document]
Comment (Commando18) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/upload/CSVUploadFileEntryHandler.java:90]: I'll sort arguments for sure

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ImportCSVMVCActionCommand.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ivicac) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ImportCSVMVCActionCommand.java:160]: @Commando18 ok, thx

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
@ivicac please review my changes, thanks!

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by brianchandotcom** [comment]
@Commando18 

1.) dependencies/testFailCSVImport.csv

should be _ and not camel cased.

2.) Better yet, just get rid of those 2 CSV files and embed the strings in the test.

3.) What is TBC ?

4.) CommerceOrderImporterTypeTest do we really need to make another company?

5.) addCatalogBaseCommercePriceList , use full name please. There's no CatalogBase

6.)

git grep "SessionErrors.get(renderRequest, \""

We rarely rarely use this pattern. In general, for errors, it's an exception. Avoid a string.

7.) This PR is kind of big.......... and it's over a week worth of work. We really need to chunk these down smaller to 1 unit of work (1 day of work)

@ivicac @Luizcarlosqueiroz fyi

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**